sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -100,27 +117,43 @@ export interface AzureMachineLearningSkill extends BaseSearchIndexerSkill {
 }
 
 // @public
-export interface AzureOpenAIEmbeddingSkill extends BaseSearchIndexerSkill {
-    apiKey?: string;
-    authIdentity?: SearchIndexerDataIdentity;
-    deploymentId?: string;
+export interface AzureMachineLearningVectorizer extends BaseVectorSearchVectorizer {
+    aMLParameters?: AzureMachineLearningVectorizerParameters;
+    kind: "aml";
+}
+
+// @public
+export type AzureMachineLearningVectorizerParameters = NoAuthAzureMachineLearningVectorizerParameters | KeyAuthAzureMachineLearningVectorizerParameters | TokenAuthAzureMachineLearningVectorizerParameters;
+
+// @public (undocumented)
+export interface AzureOpenAIEmbeddingSkill extends BaseSearchIndexerSkill, AzureOpenAIParameters {
+    dimensions?: number;
     odatatype: "#Microsoft.Skills.Text.AzureOpenAIEmbeddingSkill";
-    resourceUri?: string;
 }
 
+// @public
+export type AzureOpenAIModelName = string;
+
 // @public
 export interface AzureOpenAIParameters {
     apiKey?: string;
     authIdentity?: SearchIndexerDataIdentity;
     deploymentId?: string;
+    modelName?: AzureOpenAIModelName;
     resourceUri?: string;
 }
 
 // @public
-export type AzureOpenAIVectorizer = BaseVectorSearchVectorizer & {
-    kind: "azureOpenAI";
+export interface AzureOpenAIVectorizer extends BaseVectorSearchVectorizer {
     azureOpenAIParameters?: AzureOpenAIParameters;
-};
+    kind: "azureOpenAI";
+}
+
+// @public (undocumented)
```

Comment:
Missing documentation

Reply:
Fixed!

Reply:
these still show as undocumented - am I  looking at an outdated diff? (entirely possible!) - could you double check and confirm? Many thanks!

Reply:
Fixed!